ansible/roles/software/ckan/catalog/ckan-app/tasks/main.yml

```diff
@@ -14,6 +14,14 @@
       - swig
       - xmlsec1
 
+- name: set REQUESTS_CA_BUNDLE
+  blockinfile:
+    path: /etc/profile
+    create: true
+    block: |
+      # https://github.com/GSA/datagov-deploy/pull/1965#issuecomment-667562671
+      REQUESTS_CA_BUNDLE=/etc/ssl/certs/ca-certificates.crt
```

Comment:
Hmmm, I think my comment got lost. We might want to limit this to sandbox/staging since it is not needed in production. This is what we do in [pycsw](https://github.com/GSA/datagov-deploy-pycsw/blob/master/templates/supervisor_worker.conf.j2#L7-L10).

The reason is because the OS CA bundle can become "outdated" which is why `requests` bundles its own CAs via `certifi`.

Reply:
... And let's add a TODO comment to move this to the application environment (via .env or $HOME/profile), rather at the system level.

Reply:
`REQUESTS_CA_BUNDLE` does not work as expected. It works when user runs python code on command line, but no by ckan harvester. We have to either find a way to append the GA certificate into `certifi/cacert.pem`  so Requests module can load it, or add the `REQUESTS_CA_BUNDLE` into ckan `virtualenv` environment variable. Issue #1999 is created to address this.